### ISSUE TITLE
Peer Feedback

### ISSUE THREAD

**Issue by Holger-Harmsen** [NumEconCopenhagen/projects-2020-josephine-sofie#2]
Josephine / Sophie
Cool Parts:
- Cool introduction in which you state the purpose of your project and its phases.
- I like your #comments. Short and concise.
- Cool merging and clarifying why you’re using inner merging. 
- Nice calculating the shares of each group. Shows DataFrame-skills! :) Also relevant for what you’re trying to show of course.
- Beautiful table.
- Really nice enumerated mid-conclusions after bar chart. Nice.
- Nice index-plots. Didn’t know that feature, definitely going to use it!
- Damn cool with a drop-down.
- Damn beautiful gender charts.

Unclear Parts:
- Not sure why you define drop_these instead of just putting the list in your pop.drop-function. For ease of understanding? It looks good I guess, but could have been more efficient in terms of number of lines. You use this method of choosing data through the assignment. 
    - If you want to delete all fields not having data, you can use df.dropna(inplace=True), if that’s your problem. Then you don’t have to specify all the data actually having values before analyzing. 
- Your dropping of ancestry from dataset under your graphical analysis seems really tedious. Couldn’t that have been made easier/more simple?

In conclusion:
The best part of the project is your very clear and rigorous walk-through of what you’re actually trying to show with introduction, conclusion and half-way observations etc. Clear what your scope is. Also beautiful graphs.
The hardest part to understand was your DF-code; why choose all the data that had values instead of doing an inner merging of your data and thereafter deleting alle NA’s by df.dropna()? If I understand your code correctly (I might not!), then this would seem to save you a lot of trouble. 
It would be nice with more simple code (see point above).
An improvement would be simpler code, maybe also some descriptive inference on your data to support your charts.
Cool project! Wish I had your graphing skills!
